### ISSUE TITLE
Convert runtime files into YAML

### ISSUE THREAD

**Issue by semmerson** [Unidata/LDM#68]
Files ldmd.conf, pqact.conf, pqsurf.conf, registry.xml

**Comment by mustbei** [comment]
As to the registry.xml the new ldmadmin (Python) already parses it seamlessly. Should it be included in the conversion?

**Comment by semmerson** [comment]
What's crucial is whether or not a programmatic library exists for parsing the particular configuration-file. One exists for XML, so "registry.xml" could be left alone. Parsers for the other configuration-files, however, are hand-coded.

**Comment by mustbei** [comment]
Indeed, I had once a syntactic error in the ldmd.conf file that the current yy parser could not point me to. 
Will do for the all .conf files, acknowledging however the impact on their current usage in both the new ldmadmin and the underlying C code (pqact, etc.), as well as their backward incompatibility.

**Comment by mustbei** [comment]
Oh, and on the user... (the mandatory learning curve and the subsequent ease of use and robustness thereafter)

**Comment by semmerson** [comment]
This issue is very low priority because 1) the amount of effort is
significant; 2) user's are used to the current solution; 3) providing
forward and rollback compatibility is very non-trivial (the
configuration-files would have to be automatically converted to the new
format and changes to the new configuration-files would have to be rolled
back into the old configuration-files); and 4) there are, doubtless, more
important issues.

So, don't work on it if you were planning too.

This issue was more aspirational (i.e., wishful thinking) on my part.

**Comment by mustbei** [comment]
Makes sense. Should it be closed then?

**Comment by semmerson** [comment]
Leave it open. It's aspirational.

Hope is good.

**Comment by mustbei** [comment]
Absolutely!
